docker/zarr-to-loom/ss2_plate_zarr_to_loom.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env  python3
+
+import os
+import sys
+import zarr
+import loompy
+import scipy as sc
+from scipy.sparse import coo_matrix
+import argparse
+import numpy as np
+
+
+def main():
+    description = """This script converts SS2 pipeline zarr output into loom"""
+    parser = argparse.ArgumentParser(description=description)
+    parser.add_argument('--input-zarr', dest='input_zarr_path', required=True, help="Path to input zarr file")
+    parser.add_argument('--output-loom', dest='output_loom_path', required=True, help="Path to output loom file")
+    parser.add_argument('--sample-id', dest='sample_id', required=True, help="Sample identifier")
+    args = parser.parse_args()
+
+    # Do input checks
+    if not os.path.isdir(args.input_zarr_path):
+        sys.exit("Error: the input zarr path is not a directoyr")
+    if os.path.exists(args.output_loom_path):
+        sys.exit("Error: the output loom file exists")
```

Comment:
optional nit: use `pathlib` over `os.path` in Python3 for better safety purposes. Also output the `args. input_zarr_path` and `args. output_loom_path` for easier debugging.